clientv3/concurrency/election.go

```diff
@@ -105,6 +105,47 @@ func (e *Election) Campaign(ctx context.Context, val string) error {
 	return nil
 }
 
+// Sometimes we need to know that we will wait for a while, before node became leader
```

Comment:
Can you please explain more about why this is needed?  What issue does this try to fix?